feat(theme): Make agnoster support conda environment

AYO-YO opened the pull request:
## Standards checklist:

<!-- Fill with an x the ones that apply. Example: [x] -->

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- Add a agnoster subtheme on conda supported

## Other comments:

effect:

(Sorry, I can't upload pictures right now...)

such as:

```shell
 zcx@zcx-workpc  🐍 ecloud  ~/.oh-my-zsh/themes   master  conda deactivate
 zcx@zcx-workpc  🐍 base  ~/.oh-my-zsh/themes   master  conda deactivate
 zcx@zcx-workpc  ~/.oh-my-zsh/themes   master  vim ../.zshrc
```

AlfredMoore commented:
> LGTM! I reverted to the original order, to avoid disruption to other users.

Has been disrupted. Any extra symbol for `agnoster` is not so necessary. Could we roll it back? It might hurt many of the `agnoster` lover.

AYO-YO commented:
> > LGTM! I reverted to the original order, to avoid disruption to other users.
> 
> Has been disrupted. Any extra symbol for `agnoster` is not so necessary. Could we roll it back? This feature might hurt many of the `agnoster` lover.
> 
> It could be an option, but would better not be a default.

Do you mean emoji 🐍? Sorry for the trouble. I thought relying solely on color might not be intuitive enough to indicate what this component represents, especially in environments where both virtualenv and conda are used. That's why I added the 🐍, which resembles the conda logo.

As you rightly pointed out, I should make this an option. 🙂

AlfredMoore commented:
Thanks for understanding.

adamaps commented:
> You can execute the command `conda config --set changeps1 False` to disable conda's own prompt display. After executing the command, reload your user profile to make it take effect.
> 
> Example: ![image](https://private-user-images.githubusercontent.com/64309671/454646824-48e9d239-5451-42c8-a0e3-c90e351c2b4e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDk4MjY2ODIsIm5iZiI6MTc0OTgyNjM4MiwicGF0aCI6Ii82NDMwOTY3MS80NTQ2NDY4MjQtNDhlOWQyMzktNTQ1MS00MmM4LWEwZTMtYzkwZTM1MWMyYjRlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA2MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwNjEzVDE0NTMwMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUwZTFiZmMxMzBhY2FlYTc0ZDBiYzhmOTkwY2IxOGExM2U4YzdiNjJjZTZiNjBiMjQ1NTUxOGY5ZjNkNGY3YTAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.jbEvCmEBu8co5nF9DIHPxyi1LuLQiS29ZGwwpOuQSUI)

@AYO-YO This worked perfectly. Thank you! 🚀
